### PR TITLE
Drive by api

### PR DESCRIPTION
**Pull request by llimon**
@samkeet Do you have a quick way to test this change?

### CODE REVIEW

File: server/KubeJob.go [hunk 1]
```diff
@@ -76,14 +92,10 @@ func GetJobStatus(w http.ResponseWriter, r *http.Request) {
 			res["LastProbeTime"] = strings.Replace(jobStatus.CompletionTime.String(), "T", " ", 1)[:20]
 		}
 
-		if jobStatus.Active == 0 &&
-			jobStatus.Failed == 0 &&
-			jobStatus.Succeeded != 0 {
+		if jobStatus.Active == 0 && jobStatus.Failed == 0 && jobStatus.Succeeded != 0 {
```
Comment (samkeet) [inline server/KubeJob.go:95]: This makes line length > 80. Hence I had to keep the previous fomat

File: server/KubeJob.go [hunk 2]
```diff
@@ -76,14 +92,10 @@ func GetJobStatus(w http.ResponseWriter, r *http.Request) {
...
 			res["JobStatus"] = "Completed"
 			res["Logs"] = GetLogs(myjob)
-		} else if jobStatus.Active == 0 &&
-			jobStatus.Failed != 0 &&
-			jobStatus.Succeeded == 0 {
+		} else if jobStatus.Active == 0 && jobStatus.Failed != 0 && jobStatus.Succeeded == 0 {
```
Comment (samkeet) [inline server/KubeJob.go:98]: This makes line length > 80. Hence I had to keep the previous fomat

**Review by samkeet — APPROVED** [review]
Looks good 👍

**Comment by samkeet** [comment]
@llimon yes can do it now locally!

File: server/KubeJob.go (hunk 1 above)
Comment (llimon) [reply server/KubeJob.go:95]: is that a go standard?
over time I have come to prefer this way as it makes it easier to read the logic. Only break is is longer than 3 conditions.

Comment (samkeet) [reply server/KubeJob.go:95]: It's not a go standard. I prefer keeping line lengths < 80 chars for readability

**Pull request merged**